src/components/structures/RoomStatusBar.js

```diff
@@ -20,6 +20,8 @@ var dis = require("../../dispatcher");
 var WhoIsTyping = require("../../WhoIsTyping");
 var MatrixClientPeg = require("../../MatrixClientPeg");
 
+const HIDE_DEBOUNCE_MS = 2000;
```

Comment:
i'd guess we want to wait at least 10s before the statusbar gets bored and hides itself again, but will have a play with it first...